docs/cicd/github-cd-template.yml

```diff
@@ -80,10 +80,10 @@ jobs:
 
       # This step is to pack the Teams App as zip file,
       # which can be used to be uploaded onto Teams Client for installation.
-      - name: Build Teams App's package
+      - name: Package your Teams App for publishing
```

Comment:
Nit: a little bit weird to see `your` in the action name.

Reply:
Agree, `Package Teams App for publishing` might be better.

Reply:
@Alive-Fish For awareness, borrow it from cli's help doc. I will update the yml soon.